packages/@webex/plugin-meetings/test/unit/spec/stats-analyzer/index.js

```diff
@@ -1,2147 +1,4 @@
-
-          for (const data of [
-            mqeData.audioTransmit,
-            mqeData.audioReceive,
-            mqeData.videoTransmit,
-            mqeData.videoReceive,
-          ]) {
-            assert.strictEqual(data[0].common.common.multistreamEnabled, false);
-          }
-        });
-
-        it('false if StatsAnalyzer initialized with false', async () => {
-          statsAnalyzer = new StatsAnalyzer({
-            config: initialConfig,
-            receiveSlotCallback: () => receiveSlot,
-            networkQualityMonitor,
-            isMultistream: false,
-          });
-          registerStatsAnalyzerEvents(statsAnalyzer);
-          await startStatsAnalyzer({pc, statsAnalyzer, mediaStatus: {expected: {receiveVideo: false}}});
-
-          await progressTime();
-
-          for (const data of [
-            mqeData.audioTransmit,
-            mqeData.audioReceive,
-            mqeData.videoTransmit,
-            mqeData.videoReceive,
-          ]) {
-            assert.strictEqual(data[0].common.common.multistreamEnabled, false);
-          }
-        });
-
-        it('true if StatsAnalyzer initialized with multistream', async () => {
-          statsAnalyzer = new StatsAnalyzer({
-            config: initialConfig,
-            receiveSlotCallback: () => receiveSlot,
-            networkQualityMonitor,
-            isMultistream: true,
-          });
-          registerStatsAnalyzerEvents(statsAnalyzer);
-          await startStatsAnalyzer({pc, statsAnalyzer, mediaStatus: {expected: {receiveVideo: true}}});
-
-          await progressTime();
-
-          for (const data of [
-            mqeData.audioTransmit,
-            mqeData.audioReceive,
-            mqeData.videoTransmit,
-            mqeData.videoReceive,
-          ]) {
-            assert.strictEqual(data[0].common.common.multistreamEnabled, true);
-          }
-        });
-      });
-
-      describe('CPU Information Reporting', async () => {
-        let getNumLogicalCoresStub;
-
-        beforeEach(async () => {
-          getNumLogicalCoresStub = sinon.stub(CpuInfo, 'getNumLogicalCores');
-        });
-
-        afterEach(() => {
-          getNumLogicalCoresStub.restore();
-        });
-
-        it('reports 1 of logical CPU cores when not available', async () => {
-          getNumLogicalCoresStub.returns(undefined);
-          await startStatsAnalyzer({pc, statsAnalyzer, mediaStatus: {expected: {receiveVideo: true}}});
-
-          await progressTime();
-          assert.equal(mqeData.intervalMetadata.cpuInfo.numberOfCores, 1);
-        });
-
-        it('reports the number of logical CPU cores', async () => {
-          getNumLogicalCoresStub.returns(12);
-          await startStatsAnalyzer({pc, statsAnalyzer, mediaStatus: {expected: {receiveVideo: true}}});
-
-          await progressTime();
-          assert.equal(mqeData.intervalMetadata.cpuInfo.numberOfCores, 12);
-        });
-      });
-    })
-  });
+describe.skip('plugin-meetings', () => {
```

Comment:
I have kept this file for the work test run command for now. Otherwise, deleing this now causes window `ReferenceError: window is not defined.`
It will be fixed separately

See details here webexteams://im?space=f2606c60-8ab2-11ec-be25-b7a3d486240d&message=bc2b0750-5ba6-11ef-994e-ad1d96cf0b70